third-party/re2/CMakeLists.txt

```diff
@@ -12,3 +12,4 @@ set(RE2_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src" CACHE STRING "RE2 include
 # util/arena.h
 SET_PROPERTY(DIRECTORY PROPERTY INCLUDE_DIRECTORIES "${RE2_INCLUDE_DIR}")
```

Comment:
this would normally be redundant after the change below, but probably still needed because of what the comment says